readthedocs/templates/projects/notifications/deprecated_config_file_used_email.html

```diff
@@ -0,0 +1,10 @@
+{% extends "core/email/common.html" %}
+{% block content %}
+Your project "{{ project.slug }}" is not using a <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">.readthedocs.yaml configuration file</a> and <strong>will stop working soon</strong>.
```

Comment:
Missed this last time, but "will stop working" sounds like documentation will go offline

```suggestion
Your project "{{ project.slug }}" is not using a <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">.readthedocs.yaml configuration file</a> and <strong>builds will stop working soon</strong>.
```